build/ghost-buster/start.bash

```diff
@@ -45,6 +51,6 @@ while inotifywait -e modify /ghost/content/data/ghost.db; do sleep 10 && /ghost/
 chown -R ghost:ghost /data /ghost /ghost-override
 su ghost << EOF
 cd "$GHOST"
-NODE_ENV=${NODE_ENV:-production} npm start
+ghost run
```

Comment:
Now start command is just `ghost run`. (Command `ghost start` run process in background, but `ghost run` in foregroud)